From 2e845bbbe655e16aaa3ff53709fa6e1ba92813ab Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Tue, 14 Jan 2020 11:20:26 -0800
Subject: [PATCH 0190/1851] v4l: xilinx: Kconfig: Add missing DMADEVICES
 dependencies

commit b1091d5e0ed5a3f53ecaf89e42607c75d069dfbf from
https://github.com/Xilinx/linux-xlnx.git

Several drivers selecting Xilinx DMA drivers but not all dependencies
are properly covered.

WARNING: unmet direct dependencies detected for XILINX_FRMBUF
  Depends on [n]: DMADEVICES [=n]
  Selected by [y]:
  - VIDEO_XILINX [=y] && MEDIA_SUPPORT [=y] && V4L_PLATFORM_DRIVERS [=y] && VIDEO_V4L2 [=y] && VIDEO_V4L2_SUBDEV_API [=y] && OF [=y] && HAS_DMA [=y]

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/platform/xilinx/Kconfig b/drivers/media/platform/xilinx/Kconfig
index a2773ad7c185..7517e7046205 100644
--- a/drivers/media/platform/xilinx/Kconfig
+++ b/drivers/media/platform/xilinx/Kconfig
@@ -4,6 +4,7 @@ config VIDEO_XILINX
 	tristate "Xilinx Video IP (EXPERIMENTAL)"
 	depends on VIDEO_V4L2 && VIDEO_V4L2_SUBDEV_API && OF && HAS_DMA
 	select VIDEOBUF2_DMA_CONTIG
+	select DMADEVICES
 	select V4L2_FWNODE
 	help
 	  Driver for Xilinx Video IP Pipelines
-- 
2.31.1

